fakes/fake_env_vars.go

```diff
@@ -49,17 +49,32 @@ const Services string = `[
         {
           "id": "4bc59b9a-8520-409f-85da-1c7552315863",
           "description": "Google Cloud SQL is a fully-managed MySQL database service",
-          "name": "google-cloudsql",
+          "name": "google-cloudsql-mysql",
           "bindable": true,
           "plan_updateable": false,
           "metadata": {
-            "displayName": "Google CloudSQL",
+            "displayName": "Google CloudSQL MySQL",
             "longDescription": "Google Cloud SQL is a fully-managed MySQL database service",
             "documentationUrl": "https://cloud.google.com/sql/docs/",
             "supportUrl": "https://cloud.google.com/support/",
             "imageUrl": "https://cloud.google.com/_static/images/cloud/products/logos/svg/sql.svg"
           },
-          "tags": ["gcp", "cloudsql"]
+          "tags": ["gcp", "cloudsql", "mysql"]
+        },
+        {
+          "id": "cbad6d78-a73c-432d-b8ff-b219a17a803a",
+          "description": "Google Cloud SQL is a fully-managed PostgreSQL database service",
```

Comment:
having different DBMSs be different services is consistent with the AWS Broker, and the plans are not the same.